### PR TITLE
Apple fixes

### PR DESCRIPTION
**Pull request by adriweb**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -130,5 +147,16 @@ int main(int argc, char **argv)
...
     });
 
-    return app.exec();
+    int ret = app.exec();
+
+    #ifdef IS_IOS_BUILD
+        // New in iOS 13: due to some kernel/system bug, if we leave a process
+        // with PT_TRACE_ME, it will not get terminated properly and will refuse
+        // to launch again.
+        syscall(SYS_ptrace, 31 /* PT_DENY_ATTACH */, 0, NULL, 0);
```
Comment (Vogtinator) [inline main.cpp]: This should be done in `translate_deinit()`

Comment (adriweb) [reply main.cpp]: done

**Comment by adriweb** [comment]
Confirmed to work fine on latest iOS and macOS now 🎉

File: core/translate_arm.cpp [hunk 2]
```diff
@@ -13,6 +13,8 @@
 #include <cstdio>
 
 #ifdef IS_IOS_BUILD
+#include <unistd.h>
```
Comment (Vogtinator) [inline core/translate_arm.cpp]: `SYS_ptrace` is already used in a different place, so adding more includes shouldn't be necessary

File: core/translate_aarch64.cpp [hunk 3]
```diff
@@ -30,6 +30,8 @@
 #include "os/os.h"
 
 #ifdef IS_IOS_BUILD
+#include <unistd.h>
```
Comment (Vogtinator) [inline core/translate_aarch64.cpp]: SYS_ptrace is already used in a different place, so adding more includes shouldn't be necessary

**Pull request merged**